beta/docker/README.md

```diff
@@ -384,3 +384,19 @@ docker stack config \
     --compose-file ./compose.tls.yaml |
         docker stack deploy --compose-file - op-scim-bridge
 ```
+
+### Custom Redis Options
```

Comment:
Since these are set in [compose.template.yaml](https://github.com/1Password/scim-examples/blob/15f1a6c7361e7de0f7a78fe4868487b18a849173/beta/docker/compose.template.yaml), we should decide if we want to include them as commented-out values in the yaml, or if it is sufficient to document them here and have people amend the yaml if required. 

I'm tempted to leave them out of the yaml to keep things clean and ensure that things are not confusing for people, since the majority of people will not be using these settings.